### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by jonasgeiler** [jonasgeiler/lua-fenster#14] [Announcements]
I am thrilled to announce the release of version 1.0.0, the first major release of lua-fenster! 🥳 
This marks the transition of this library to a stable state. Any API changes in the future will be more or less backward compatible and new features or deprecations will be introduced in a more graceful way.

Have fun! 🎉 

## What's Changed
* The library is now "stable"
* feat: add rhodonea curve demo by @jakob-kruse in https://github.com/jonasgeiler/lua-fenster/pull/12
* chore(deps): update actions/checkout action to v4.1.4 by @renovate in https://github.com/jonasgeiler/lua-fenster/pull/13

**Full Changelog**: https://github.com/jonasgeiler/lua-fenster/compare/v0.4.0...v1.0.0

## New Contributors
* @jakob-kruse made their first contribution in https://github.com/jonasgeiler/lua-fenster/pull/12

<hr /><em>This discussion was created from the release <a href='https://github.com/jonasgeiler/lua-fenster/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by jonasgeiler** [comment]
@pauladam94 I've finally released v1.0.0! Just letting you know.

**Reply by pauladam94** [reply]
Nice I will go see that.
